cmd/root.go

```diff
@@ -78,15 +78,15 @@ var RootCmd = &cobra.Command{
 			stdout.Writer = colorable.NewNonColorable(os.Stdout)
 			stderr.Writer = colorable.NewNonColorable(os.Stderr)
 		}
-		golog.SetOutput(log.StandardLogger().Writer())
+		logrus.SetOutput(logrus.StandardLogger().Writer())
```

Comment:
I am pretty sure this still needs to use the std log otherwise we won't redirect it's output to the logrus one

Reply:
@MStoykov nice catch, fixed.